src/main/java/no/fdk/model/fuseki/action/CompactAction.java

```diff
@@ -16,18 +20,27 @@
 import java.util.stream.Stream;
 
 @Slf4j
+@Component
 public class CompactAction extends BaseActionREST {
     @Override
     protected void doGet(HttpAction action) {
-        log.info("Started compaction process");
+        doCompact(action.getDataset());
+    }
 
-        DatasetGraph datasetGraph = action.getDataset();
+    private void doCompact(DatasetGraph datasetGraph) {
```

Comment:
Rename til compact. doGet måtte til fordi det overrider metode i base klasse

```suggestion
    private void compact(DatasetGraph datasetGraph) {
```